### DISCUSSION TITLE
Snap apps and Fcitx5 keyboard

### DISCUSSION THREAD

**Discussion by morti86** [WayfireWM/wayfire#2415] [Q&A]
Like in the title, I have two questions, or even one more:
1. How to see installed Snap apps? I cannot see any of them in the launcher.
2. How to make fcitx5 launch with Wayfire automatically
3. Is there a way to control the look of KDE/Qt apps (they start with the default theme)?

**Comment by ammen99** [comment]
I can only really help with 2), you can set fcitx5 in your autostart. Something like this:

```ini
[autostart]
fcitx5 = sleep 2 && fcitx5
```

About qt, you can use any of the standard qt tools. Personally I haven't used them but it should work.

About snap apps: wf-shell uses standard GTK methods to list the available applications, so I am not sure why snap packages are not being picked up. Maybe it is an environment variable that you are missing? But I do not know which ones are important ...
